recipes/pam.rb

```diff
@@ -71,7 +72,7 @@
   end
 
   # configure tally2
-  if node['os-hardening']['auth']['retries'] > 0
+  if node['os-hardening']['auth']['retries'].positive?
```

Comment:
@atomic111 fixed. chef 12.5.1 has ruby 2.1 and not 2.3 (2.1 does not have `positive?`), I changed the rubocop settings to reflect that and also created https://github.com/dev-sec/chef-ssh-hardening/pull/179